paket.dependencies

```diff
@@ -15,33 +15,33 @@ source https://api.nuget.org/v3/index.json
 // nuget Microsoft.AspNetCore.All 2.0.0
 
 // DOTNETCORE & PAKET ISSUES
-nuget Microsoft.NETCore.App 2.2.0
-nuget Microsoft.NETCore.DotNetAppHost 2.2.0
-nuget Microsoft.NETCore.DotNetHostResolver 2.2.0
-nuget Microsoft.NETCore.DotNetHostPolicy 2.2.0
+nuget Microsoft.NETCore.App 2.2.4
+nuget Microsoft.NETCore.DotNetAppHost 2.2.4
+nuget Microsoft.NETCore.DotNetHostResolver 2.2.4
+nuget Microsoft.NETCore.DotNetHostPolicy 2.2.4
 nuget NETStandard.Library 2.0.3
 
 // PRODUCTION STUFF
 nuget Microsoft.Extensions.Configuration 2.2.0
 nuget Microsoft.Extensions.Configuration.CommandLine 2.2.0
 nuget Microsoft.Extensions.Configuration.FileExtensions 2.2.0
-nuget Microsoft.Extensions.Configuration.EnvironmentVariables 2.2.0
+nuget Microsoft.Extensions.Configuration.EnvironmentVariables 2.2.4
 nuget Microsoft.Extensions.Configuration.Json 2.2.0
 
-nuget Autofac 4.9.1
```

Comment:
You will also need:
https://github.com/Informatievlaanderen/templates/blob/master/src/Be.Vlaanderen.Basisregisters.Templates/Be.Vlaanderen.Basisregisters.Registry/paket.dependencies#L29-L30

```
// For more healtchecks, look at https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks
nuget AspNetCore.HealthChecks.SqlServer 2.2.0
```